code/modules/paperwork/stamps.dm

```diff
@@ -12,6 +12,68 @@
 	item_color = "cargo"
 	pressure_resistance = 2
 	attack_verb = list("stamped")
+	/// used for on human stamp marks, each mark will be colored this before adding to image in human stamp_marks
+	var/stamp_color = "#6ca151"
+	/// this is defined uniquely for each subtype, used in examine.dm for when humans are stamped.
+	var/stamp_description = "..."
+
+/obj/item/stamp/afterattack(atom/target, mob/user, proximity, params)
+	. = ..()
+	if(!ishuman(target) || !ishuman(user) || !proximity)
+		return
+
+	var/mob/living/carbon/human/H = target
+
+	if(length(H.stamp_marks.overlays) >= 20)	//no more than 50 marks on a person, otherwise infinite laggy stamp marks
+		to_chat(user, "<span class='notice'>Bluespace mechanics prevent any more marks from appearing on [target.name].</span>")
```

Comment:
1) Comment was mismatched from the code
2) IMO a reason other than "Bluespace mechanics" would be nice here. But not a huge deal either way.
```suggestion
	if(length(H.stamp_marks.overlays) >= 20)	//no more than 20 marks on a person, otherwise infinite laggy stamp marks
		to_chat(user, "<span class='notice'>[target.name] is already completely covered in stamps!</span>")
```

Reply:
I changed it to "[target.name]'s skin is saturated with ink! No more marks can appear on them." , hopefully this is less 4th wall breaking.